libs/external-db-spanner/src/spanner_config_validator.ts

```diff
@@ -5,7 +5,7 @@ export class ConfigValidator implements IConfigValidator {
     requiredKeys: string[]
     config: any
     constructor(config: any) {
-        this.requiredKeys = ['projectId', 'instanceId', 'databaseId', 'secretKey']
+        this.requiredKeys = ['projectId', 'instanceId', 'databaseId', 'externalDatabaseId', 'allowedMetasites']
```

Comment:
@rosenvered, I noticed it just now, 
There's a reason that only in spanner config validator specifically there is `externalDatabaseId, allowedMetasites` ? 
there was `secretKey`, but it was by mistake and I fixed it on master.